packages/r/radiotray-ng/abi_libs

```diff
@@ -1,2 +1 @@
-radiotray-ng
```

Comment:
This is kinda alarming. Can you test `radiotray-ng` and make sure it behaves normally?

Reply:
Works fine. Listened to smooth jazz.